base/deprecated.jl

```diff
@@ -259,3 +257,9 @@ const base64 = base64encode
 
 @deprecate sizehint(A, n) sizehint!(A, n)
 
+@deprecate randbool!             rand!
+@deprecate randbool()            rand(Bool)
+@deprecate randbool(dims)        bitrand(dims)
+@deprecate randbool(dims...)     bitrand(dims)
+@deprecate randbool(r, dims)     bitrand(r, dims)
+@deprecate randbool(r, dims...)  bitrand(r, dims)
```

Comment:
Do we not have `rand(r::AbstractRNG, t::Type)` so that different RNGs can be used to generate random values of various integer/boolean types? It appears not.